migration_conf.js

```diff
@@ -1,8 +1,6 @@
 const path = require("path")
 
-// eslint-disable-next-line no-undef
 const BUILD_DIR = path.join(__dirname, "build", "contracts")
-// eslint-disable-next-line no-undef
 const NETWORKS_FILE_PATH = path.join(__dirname, "networks.json")
```

Comment:
Weird... why where these even here. Looks like they were perfectly defined..

Reply:
Because the ESLint `env` was incorrectly setup for this file.